### PR TITLE
Remove input checking

### PR DESCRIPTION
**Pull request by em-c-rod**
Resolves: #195 
- Remove input checking with added inputs
- Remove redundant inputs
- Rename inputs to match rhel8

### CODE REVIEW

File: inspec.yml [hunk 1]
```diff
@@ -192,97 +192,52 @@ inputs:
...
   # SV-204411
-  - name: expected_difok
-    desc: Number of characters that must be different from previous password
-    type: Numeric
-    value: 8
-
-  # SV-204411
-  - name: max_difok
+  - name: difok
     desc: Minimum number of characters that must be different from previous password, by policy
```
Comment (Hookwitz) [inline inspec.yml:202]: This desc confuses me for some reason, the original was for `max_difok` and the check in the SV-204411 control is for lte, should this not read Maximum number ... ?

File: controls/SV-204421.rb [hunk 2]
```diff
@@ -33,8 +33,7 @@
       next unless user(user).uid >= 1000
 
       describe shadow.users(user) do
-        its('max_days.first') { should cmp input('expected_max_password_lifetime') }
-        its('max_days.first') { should cmp <= input('max_password_lifetime') }
+        its('max_days.first') { should cmp <= input('password_lifetime') }
```
Comment (Hookwitz) [inline controls/SV-204421.rb:36]: It looks like based on the edit, this _should_ be `max_password_lifetime` given the comment above the input in the inspec.yml and this check is a lte?

**Review by Hookwitz — CHANGES_REQUESTED** [review]

File: inspec.yml (hunk 1 above)
Comment (em-c-rod) [reply inspec.yml:202]: According to the check text, the inspec.yml desc is correct, but the control was updated incorrectly in the PR. Changing the control to read:
`    its('difok') { should cmp >= input('difok') }`

File: controls/SV-204421.rb (hunk 2 above)
Comment (em-c-rod) [reply controls/SV-204421.rb]: Good catch. I renamed them so there is one min and one max.

**Review by Amndeep7 — CHANGES_REQUESTED** [review]
204406 also is exhibiting this behavior

**Review by wdower — APPROVED** [review]

**Pull request merged**